src/annotator/config/test/settings-test.js

```diff
@@ -395,4 +395,47 @@ describe('annotator/config/settingsFrom', () => {
       });
     });
   });
+
+  describe('#sideBySide', () => {
+    [
+      {
+        input: 'foo',
+        expectedResult: { mode: 'auto' },
+      },
+      {
+        input: {},
+        expectedResult: { mode: 'auto' },
+      },
+      {
+        input: { mode: 'invalid' },
+        expectedResult: { mode: 'auto' },
+      },
+      {
+        input: { mode: 'auto' },
+        expectedResult: { mode: 'auto' },
+      },
+      {
+        input: { mode: 'manual' },
+        expectedResult: { mode: 'manual' },
+      },
+    ].forEach(({ input, expectedResult }) => {
```

Comment:
Coercing the input to a legal value makes sense at present, but it would be preferable to at least log a warning in future.